### PR TITLE
ADD Contact Form Section and Footer in the portfolio

### PR DESCRIPTION
**Pull request by joramot**
The tasks to do are:

- Add Contact Form Section in index.html.
- Add Footer in index.html.
- Add CSS classes to the Contact Form
- Add CSS classes to the Footer.
- Add social media icons in the images folder.

### CODE REVIEW

**Review by alizaamir123 — CHANGES_REQUESTED** [review]
I really appreciate the work you've done so far on this design.
I noticed that the design doesn't quite match the reference in Figma. I think it would be helpful to adjust a few elements to more closely align with the reference

**Comment by joramot** [comment]
ok @alizaamir123 thanks, I will do it the changes.

**Review by Kamba56 — COMMENTED** [review]
Hi @joramot

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that **does not have contributions from your coding partner.** This Is a pair programming project, kindly work on it with your coding partner. there should be commit messages from your coding partner(s).

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by joramot** [comment]
hello @Kamba56 my coding partner @alizaamir123 did contributions with her comments https://github.com/joramot/capstone_portfolio/pull/4#pullrequestreview-1361622340 
Is not correct?

**Comment by Kamba56** [comment]
> hello @Kamba56 my coding partner @alizaamir123 did contributions with her comments [#4 (review)](https://github.com/joramot/capstone_portfolio/pull/4#pullrequestreview-1361622340) Is not correct?

Hi @joramot, this is a pair programming project. This means that you and your partner should take turns being the driver and the navigator. The driver should write the code while the navigator makes suggestions. When it's time to switch turns, the driver pushes their changes to github. So commit messages from the parties involved should be visible on the PR. To achieve this, you need to add your partner as a contributor to the repo.

**Comment by joramot** [comment]
@Kamba56  oooh!!, ok, I understand now, thanks so much @Kamba56 I will do it.

**Review by adel-gu — COMMENTED** [review]
# Status: INVALID ❌ 

**Hi @joramot 👋🏼** 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that contains the following issues:
-  The PR should contain a descriptive title and a summary that describe the achievements and the changes are been made in this milestone 👍🏼 
- This is a group project so the commit history should contain all the contributors' names for each commit according to how you split tasks among each other.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by joramot** [comment]
Hello @adel-gu thanks for your time and observations, I will do the corrections in the description of the PR, and complete the peer-to-peer activity with my partners.

**Comment by ABDULALI3468** [comment]
@joramot It's me ABDUL ALI your Code Reviewer. I understand your problem you did a great job. I am reviewing your Pull request. Just wait for a few minutes. I will get back to you very soon!

File: index.html [hunk 1]
```diff
@@ -217,6 +210,50 @@ <h2 class="about-me-title">About me</h2>
...
+      </p>
+      <form class="contact-form-mobile contact-form-desktop" action="https://formspree.io/f/mlekejwj" method="post">
+        <input type="text" id="name" name="user_name" maxlength="30" placeholder="Full name" required />
+        <hr class="separator-line">
+        <input type="email" id="mail" name="user_email" placeholder="Email address" required />
+        <hr class="separator-line">
+        <textarea class="textarea_mobile" id="msg" name="user_message" maxlength="500" placeholder="Enter text here"
+          required></textarea>
+        <input class="button-colors" type="submit" value="Get in touch"></input>
+      </form>
```
Comment (ABDULALI3468) [inline index.html:226]: **1-** Please consider making the form elements in the center horizontally to make it look alike the Figma Template.

## Figma Design:-

image


## Your Design:-

image

**2-** Moreover, please consider adding or removing the outline from all the input including the text area field to match the design. The outline comes when the user clicks on the input.

```
outline: none;
```

File: css/style.css [hunk 2]
```diff
@@ -506,3 +506,127 @@ li {
...
+  align-items: center;
+}
+
+.bottom-nav {
+  margin-top: 15px;
+  margin-bottom: 13px;
+  display: flex;
+  align-items: center;
+  gap: 2rem;
+}
```
Comment (ABDULALI3468) [inline css/style.css:622]: Please consider reducing the gap to like `20px` to make it closer to the design.

## Your Design:--

image


## Figma Design:-

image


## Solution:-

Update the gap property to following:-

```
gap: "20px";
```

**Review by ABDULALI3468 — CHANGES_REQUESTED** [review]
# Changes Required 👍 

Hi @joramot @alizaamir123 , 



Indeed you did an exceptional but still, there is room for changes to complete this project. 💯


## Highlights 
✅Your design looks great.
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.


 ## Required Changes ♻️

> **_Check the comments below to have a look at the required changes._**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shisui6 — APPROVED** [review]
Hi @joramot, 

# Approved 🪄
Your project is complete! There is nothing else to say other than... it's time to merge it 💎
Congratulations! 🎉 

## Highlights
- HTML validation done correctly ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by joramot** [comment]
@Shisui6 thanks for your time.

**Comment by adel-gu** [comment]
Hi @joramot 👋🏼, Sorry for the late replay, I gues the issue has been resolved 👍🏼.

Congratulations ✨

**Pull request merged**